beam-postgres/Database/Beam/Postgres/Extensions.hs

```diff
@@ -195,3 +195,12 @@ pgDropExtensionProvider =
      pure (PotentialAction (HS.fromList [p extP]) mempty
                            (pure (MigrationCommand cmd MigrationKeepsData))
                            ("Unload the postgres extension " <> ext) 1)
+
+-- *** Built-in extensions
+
+-- | Data type representing definitions contained in the @plpgsql@ extension
+data PlPgSql = PlPgSql
+
+instance IsPgExtension PlPgSql where
+  pgExtensionName Proxy = "plpgsql"
+  pgExtensionBuild = PlPgSql
```

Comment:
Are there any tests for this we can add?

Reply:
I realize there are no methods to this extension... which begs the question: Why have this at all?

Reply:
I needed this because I'm using a custom version of `verifySchema` which checks that there are no "extra" predicates in addition to that no expected predicates are missing. I still need to upstream that so happy to remove it from this PR if you think it's not useful.

Reply:
Hm, I don't have a strong opinion if you have plans to upstream that stuff anyway. The disadvantage of adding it now is that changing it later means major version bump whereas adding it all at once later is a minor version bump.

Reply:
I removed `plpgsql` for now. I'll put that in a separate PR when I upstream the two-way verification support.

Reply:
For reference the feature I was talking about is now in #522.